.editorconfig

```diff
@@ -20,3 +20,10 @@ insert_final_newline = false
 # trailing spaces in markdown indicate word wrap
 [*.md]
 trim_trailing_whitespace = false
+
+[*.py]
+multi_line_output = 3
+include_trailing_comma = True
+force_grid_wrap = 0
+use_parentheses = True
+ensure_newline_before_comments = True
```

Comment:
I guess these properties are documented somewhere? At least not at https://github.com/editorconfig/editorconfig/wiki/EditorConfig-Properties. But as those values are recommended at https://black.readthedocs.io/en/stable/compatible_configs.html, I guess we'll use them.

Reply:
I was also confused at first as they are also not supported by all the editors.

But they are an extension used by other tools like [isort](https://pypi.org/p/isort/).

And as far as extending the configuration does not break the support of in other tools I also thing we should keep them.